base/iostream.jl

```diff
@@ -229,6 +229,17 @@ function readline(s::IOStream)
     ccall(:jl_readuntil, Ref{String}, (Ptr{Void}, UInt8, UInt8), s.ios, '\n', 1)
 end
 
+"""
+    readuntil_string(s::IO, delim::UInt8)
+
+Like `readuntil(s, delim)`, but returns a `String` rather than
+a `Vector{UInt8}`.
+"""
+readuntil_string(s::IO, delim::UInt8) = String(readuntil(s, delim))
+function readuntil_string(s::IOStream, delim::UInt8)
```

Comment:
Okay, but that would be a breaking change.

(Though right now I can only find [one package using](https://github.com/BioJulia/BufferedStreams.jl/blob/a89539edb07e188c47abe28a36e22a4e1602763f/test/runtests.jl#L127) the `UInt8` version of `readuntil`.)